build/bin.LICENSE

```diff
@@ -1752,3 +1752,39 @@ licenses.
     The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+- extension/dist/ext/extension.js
```

Comment:
The dfdl.tmLanguage.json doesn't end up in extension.js right? This stays as a json file inside the .vsix?

Reply:
Yeah that is correct sorry that is my bad I will push that up with the squashed commit

Reply:
We do have two +1's now, but this is a large change, I would be more comfortable if some other reviewers took a look at things. Do any other committers have a few spare cycles to take a look?

Reply:
@rt320 @michael-hoke @arosien @scholarsmate Can you one or more of you guys review the PR and leave any thoughts you may have?

Reply:
👀

Reply:
@stevedlawrence We good to merge this in after squash? I believe @arosien also approved